onnxruntime/test/providers/provider_test_utils.cc

```diff
@@ -25,6 +25,18 @@ namespace onnxruntime {
 namespace test {
 
 // Check functions for tensor types
+template <typename T>
+void sort_expected_and_actual_buffers(const T* expected, const T* accepted, int64_t size) {
```

Comment:
>accepted [](start = 66, length = 8)

nit: Maybe it's just me, but 'accepted' somewhat implies the value is good. In this case it's just the output we need to validate isn't it, so either 'output' or 'actual' may be slightly better?

Reply:
yes, 'actual' seems better. Fixed.